### ISSUE TITLE
Tables + Pagination

### ISSUE THREAD

**Issue by canphaz** [javve/list.js#494]
Hi,

Is it possible to use Pagination with a Table? As far as I have been able to test this, it works fine when using a <UL> tag, but pagination doesn't work if List.JS is used in a <TABLE>.

Sorry if this is a duplicate question, I searched through open and closed issues, but found only older posts, since which List.JS has changed significantly (pagination used to be a separate plugin? etc).

Thanks!

**Comment by elsuenog** [comment]
Yes, Just add  `<ul class="pagination"></ul>` inside of you div and add `pagination: true `to your initialization.

http://codepen.io/elsuenog/pen/XMyWdO

**Comment by hsinewu** [comment]
Hi, just some reminder.

If ul.pagination isn't show up, and only showing the content of first page.

Check if you are using v1.4.1, the version used in examples (from http://listjs.com/assets/javascripts/list.min.js).

If so, switch to [v1.5.0](http://cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js) and see if the problem is resolved.

**Comment by yong0011** [comment]
http://codepen.io/elsuenog/pen/XMyWdO
This worked for me, Thanks
